resources/public/api/conf/1/application.raml

```diff
@@ -0,0 +1,35 @@
+#%RAML 1.0
+---
+
+title: Individuals
+version: 1.0
```

Comment:
Should this be just `1` without the minor version?

Reply:
again, this is related to the _raml_ file version. Not sure if it's not going to have a minor either.